homeassistant/components/sensor/manifest.json

```
@@ -3,5 +3,6 @@
   "name": "Sensor",
   "documentation": "https://www.home-assistant.io/integrations/sensor",
   "codeowners": [],
-  "quality_scale": "internal"
+  "quality_scale": "internal",
+  "dependencies": ["history"]
```

this is meh.

I think that we should either:
 - move the parts of the history API that relate to query-ing the DB to `recorder` integration
 - update `hassfest` such that having a `recorder` platform allows importing from `history` without an explicit dependency.

---

(I prefer 1)

---

OK, here's an attempt: https://github.com/home-assistant/core/pull/50287

---

That PR got merged. After rebase:

```suggestion
  "dependencies": ["recorder"]
```

---

Please rebase so that this can become
```suggestion
  "after_dependencies": ["recorder"]
```